devfiles/go-web-sample/devfile.yaml

```diff
@@ -0,0 +1,92 @@
+---
+apiVersion: 1.0.0
+metadata:
+  generateName: golang-web-
+projects:
+-
+  name: example
+  source:
+    type: git
+    location: https://github.com/xesina/golang-echo-realworld-example-app
+  clonePath: src/github.com/xesina/golang-echo-realworld-example-app
```

Comment:
why not clone this into a shorter folder path, like just `golang-echo-realworld-example-app`?

Reply:
My guess would be `$GOPATH` compatibility issues. It's a similar case for our current Go sample.

Reply:
https://github.com/xesina/golang-echo-realworld-example-app uses go modules https://blog.golang.org/using-go-modules. Are we sure we need to use $GOPATH?

Reply:
yes, it works.